ccm_web/webpack/webpack.dev.ts

```diff
@@ -0,0 +1,12 @@
+import * as webpack from 'webpack'
+import { merge } from 'webpack-merge'
+
+import commonConfig from './webpack.common'
+
+
+const devConfig: webpack.Configuration = merge(commonConfig, {
+    mode: 'development',
+    devtool: 'inline-source-map'
+})
```

Comment:
I really don't feel the source map is working since i still see compiled stuff
![Screen Shot 2021-03-05 at 2 58 01 PM](https://user-images.githubusercontent.com/8579775/110167643-06700a00-7dc4-11eb-9bcf-6138f57aec35.png)

this is minor but can be handled later on

Reply:
Hm, it's close. I think what your saying is ideally you'd be able to view TypeScript there? Or at least get an accurate line number for the TypeScript file?

Reply:
sourcemap will exactly output the file as is in IDE. The benefit of this when debugging with breakpoint it is clear and it reflect with written code than compiled code

Reply:
Working on it, hopefully won't be too difficult.

Reply:
What about just the `source-map` option?

Reply:
Should be fixed with this change: https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/24/commits/2e0ae9ddfe6133ddef63bfca3c70a7b3eece394c

We'll probably want to add a source map for production, probably the one @jonespm recommended.

Reply:
you should not add source map to production since it adds to the launch time. you webpack.config reflects that as of now. should be good

Reply:
@pushyamig, `webpack` actually says differently: https://webpack.js.org/guides/production/#source-mapping Different options seem to have different performance implications.

https://webpack.js.org/configuration/devtool/#devtool